lib/simplifyExpression/collectAndCombineSearch/ConstantOrConstantPower.js

```diff
@@ -38,7 +38,8 @@ function getExponentNode(node) {
 // Checks if the node is an constant or a power with constant base.
 // e.g. 2^3 is a constant power node, 5 is a constant node, x and x^2 are not
 function isConstantOrConstantPower(node) {
-  return ((NodeType.isOperator(node, '^') && NodeType.isConstant(node.args[0])) ||
+  return ((NodeType.isOperator(node, '^') &&
+          NodeType.isConstant(node.args[0])) ||
```

Comment:
wait why'd you change it?

I think 
```
(a and b) OR
c
```
is more clear than
```
(a and
b) OR
c
```

was it over 80 characters? it doesn't seeeem like it